static/app/views/performance/vitalDetail/utils.tsx

```diff
@@ -139,3 +140,10 @@ export function getMaxOfSeries(series: Series[]) {
   }
   return max;
 }
+
+export const vitalToMetricsField: Record<string, TransactionMetric> = {
```

Comment:
If you care about exact 1:1 mapping here, you could, but it may be an overkill here so feel free to dismiss this 
```
{ [K in keyof WebVital]: TransactionMetric[`SENTRY_TRANSACTIONS_MEASUREMENTS_${K}`] }
```

Reply:
sure.. I could do something like:

```
Object.keys(
  WebVital
).reduce((acc, key) => {
  acc[WebVital[key]] = TransactionMetric[`SENTRY_TRANSACTIONS_MEASUREMENTS_${key}`];
  return acc;
}, {});
```

but I think I like more the current solution as it is more clear